model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -0,0 +1,629 @@
+    "id": "7wR4HJu-emJP"
+   },
+   "outputs": [],
+   "source": [
+    "dicom_data_dir = \"tciaDownload\"\n",
+    "\n",
+    "# The series_uid defines their directory where the MR data was stored on disk.\n",
+    "mr_series_uid = df.at[df.Modality.eq('MR').idxmax(), 'Series UID']\n",
+    "mr_dir = os.path.join(dicom_data_dir, mr_series_uid)\n",
+    "\n",
+    "# Read the DICOM MR series' objects and reconstruct them into a 3D ITK image.\n",
+    "#   The itk.F option is added to store the image in memory using floating-point precision pixels (useful if you will filter the image or use it with MONAI).\n",
+    "#   For more info on imread, see https://itkpythonpackage.readthedocs.io/en/master/Quick_start_guide.html.\n",
+    "mr_image = itk.imread(mr_dir, itk.F)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "JdT44AzPemJP"
+   },
+   "outputs": [],
+   "source": [
+    "# The series_uid defines where the RTSTRUCT was stored on disk.  It is stored in a single file.\n",
+    "seg_series_uid = df.at[df.Modality.eq('SEG').idxmax(), 'Series UID']\n",
+    "seg_dir = os.path.join(dicom_data_dir, seg_series_uid)\n",
+    "seg_file = glob.glob(os.path.join(seg_dir, \"*.dcm\"))[0]\n",
+    "\n",
+    "# Read the DICOM SEG object using pydicom and pydicom_seg.\n",
+    "seg_dicom = pydicom.dcmread(seg_file)\n",
+    "seg_reader = pydicom_seg.MultiClassReader()\n",
+    "seg_obj = seg_reader.read(seg_dicom)\n",
+    "\n",
+    "# Convert the DICOM SEG object into an itk image, with correct voxel origin, spacing, and directions in physical space.\n",
+    "seg_image = itk.GetImageFromArray(seg_obj.data.astype(np.float32))\n",
+    "seg_image.CopyInformation(mr_image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "yGV9j8TbemJP",
+    "tags": []
+   },
+   "source": [
+    "# 4. MONAI Zoo Basics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "diJ1EeQQemJQ"
+   },
+   "outputs": [],
+   "source": [
+    "model_name = \"prostate_mri_anatomy\"\n",
+    "model_version = \"0.3.1\"\n",
+    "zoo_dir = os.path.abspath(\"./models\")\n",
+    "\n",
+    "download(name=model_name, version=model_version, bundle_dir=zoo_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "xcj8cA_ZemJQ"
+   },
+   "outputs": [],
+   "source": [
+    "# This model includes scripts that must be run on new data.\n",
+    "#    We could import those scripts into this python notebook, but they\n",
+    "#    bring in additional dependencies.   Instead, we provide the following\n",
+    "#    more compact and compatible implementation.  Otherwise, you can\n",
+    "#    include the model's script directory by uncommenting these lines and\n",
+    "#    installing their dependencies and doing appropriate data conversions.\n",
+    "# scripts_dir = os.path.join(zip_dir, model_name, \"scripts\")\n",
```

Comment:
`zip_dir` seems unreferenced. Maybe `zoo_dir`?

Reply:
I've asked Stephen Aylward to take a look at this discussion.  I'm not sure what he intended here.

Reply:
I've changed that to zoo_dir for now, please feel free to revise.